test/TestHelper.h

```diff
@@ -50,6 +50,8 @@ void mine(BlockHeader& _bi, SealEngineFace* _sealer, bool _verify = true);
 namespace test
 {
 
+const int c_testHomesteadBlock = 900000;
```

Comment:
According to our last call, I think that needs to be changed back to 1.000.000 (also update the test accordingly)